docker-compose.yml

```diff
@@ -1,8 +1,107 @@
 ---
-version: "3.7"
+version: '2'
```

Comment:
Why version 2? and why we need so much docker-compose files?

Reply:
I consolidated all dockerfiles in one

Reply:
there are still a lot of docker-compose files, please remove other ones if they are not required or add comment why they are needed.

Reply:
done